app/templates/Mail_Donation_Confirmation.twig

```diff
@@ -1 +1,90 @@
-TODO
\ No newline at end of file
+{% set formattedAmount = donation.amount|localizedcurrency('EUR', 'de_DE') %}
+{$ recipient.salutation $}
+
+wie uns liegt Ihnen Wikipedia am Herzen. Sie engagieren sich mit einer
+Spende, damit Wikipedia weiterhin besteht und Millionen Menschen auch in Zukunft
+freien Zugang zu Wissen haben. Dafür danke ich Ihnen herzlich.
+
+{% if donation.needsModeration %}
+Ihre Spende in Höhe von {$ formattedAmount $} wurde zur Überprüfung vorgemerkt.
+Bevor wir Ihre Spende abbuchen, werden wir Sie noch einmal kontaktieren.
+
+{% else %}
+{% if donation.paymentType == 'BEZ' %}
+In Kürze erhalten Sie eine Vorankündigung Ihres SEPA-Lastschriftmandats. Dort ist auch Ihre Mandatsreferenznummer enthalten.
+Ihre Spende in Höhe von {$ formattedAmount $} werden wir im Anschluss von dem von Ihnen angegebenen Konto abbuchen.
+{% if donation.interval > 0 %}
+{$ 'recurringDonation'|transchoice( donation.interval, { '%interval%': donation.interval } ) $}
+{% endif %}
+
+{% elseif donation.paymentType == 'PPL' or donation.paymentType == 'MCP' %}
+Ihre Spende in Höhe von {$ formattedAmount $} wurde durch unseren Partner {$ donation.paymentType|trans $} gebucht.
```

Comment:
This should be `{$ donation.paymentType|trans({}, 'paymentTypes') $}` otherwise the payment type will be looked up in the default messages file.